vulture.py

```diff
@@ -394,6 +395,10 @@ def csv(option, _, value, parser):
 
 def main():
     options, args = parse_args()
+    whitelist_path = os.path.abspath(stdlib.__file__)
+    if whitelist_path.endswith('.pyc'):
+        whitelist_path = whitelist_path[:-1]
+    args.append(whitelist_path)
```

Comment:
> As a side effect, users can now disable default whitelist files by using --exclude whitelists since "whitelists" is part of the path name.

That's a win-win 🎉